qemu/tests/cfg/pktgen.cfg

```diff
@@ -19,7 +19,6 @@
             image_snapshot = yes
         - host_guest:
             #host as server
-            password_pktgen_server = redhat
```

Comment:
It should be operational without our internal configuration, so if this parameter must exist for the test case? If so, at least should keep a default value here.

Reply:
If we need a default value here, then it should be set to 'kvmautotest' instead of 'redhat'.
So modify it sounds ok to you?

Reply:
Yep

Reply:
I'd thing put it this way is better:
`# password_pktgen_server = <your_password>`
It make sense for upstream users, as they probably will different pwds.

Reply:
you may need to add an additional comement to tell users to uncomment that line (`# password_pktgen_server = <your_password>`) and add their own pwd.

Reply:
Updated. Thanks.